ehsm_kms_service/kubernetes/readme.md

```diff
@@ -0,0 +1,83 @@
+Step 1 Installing the NFS Server
+
+    1. NFS server
+    
+        yum install -y nfs-common nfs-utils rpcbind
+        mkdir /intel-ehsm-db
+        chmod 777 /intel-ehsm-db
+        chown nfsnobody /intel-ehsm-db
+        
+        cat > /etc/exports <<EOF 
+        /intel-ehsm-db *(rw,no_root_squash,no_all_squash,sync) 
+        EOF
+        
+        systemctl start rpcbind & systemctl enable rpcbind
+        systemctl start nfs & systemctl enable nfs
+        
+    2. All Kubernetes node
+    
+        yum -y install nfs-utils recbind

```

Comment:
recbind ?

Reply:
not sure whether need to start rpcbind here?

Reply:
typo, rpcbind.

Reply:
k8s pv needs it